.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,17 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag my-image-name:$(date +%s)
```

Comment:
this needs an image name, doesn't it?

Reply:
Not sure I thought `docker build .` is enough

Reply:
well you use `--tag` here and that should have a good image name

Reply:
Ah yes you are right